### PR TITLE
SDK-445: Add ssl support for Docker container

### PR DESCRIPTION
**Pull request by sidibos**

### CODE REVIEW

File: yoti/views/admin-options.php [hunk 1]
```diff
@@ -15,6 +15,7 @@
     <h1>Yoti Settings</h1>
     <p>You need to first create a Yoti App at <?php echo $dashboardLink ?>.</p>
     <p>Note: On the Yoti Dashboard the callback URL should be set to: <code><?php echo site_url('wp-login.php?yoti-select=1&action=link', 'https'); ?></code></p>
+    <p>Warning: User IDs provided by Yoti are only valid within the scope of the application. Changing your Yoti application will result to different Yoti user IDs.</p>
```
Comment (echarrod) [inline yoti/views/admin-options.php]: will result **in** different Yoti user IDs.

**Review by echarrod — APPROVED** [review]
All good except for that one word change

File: README.md [hunk 2]
```diff
@@ -45,35 +46,36 @@ Please do not open the .pem file as this might corrupt the key and you will need
...
 
 Build the containers:
 
 ```shell
-docker-compose up -d
+$ docker-compose up -d
 ```
 
-After the command has finished running, go to [http://localhost:7000](http://localhost:7000) and follow the instructions.
+After the command has finished running, go to [https://localhost:7001](https://localhost:7001) and follow the instructions.
```
Comment (bucky-boy) [inline README.md:78]: Again, why do we use a different port for every SDK?

File: docker/Dockerfile [hunk 3]
```diff
@@ -66,10 +70,12 @@ RUN set -ex; \
...
 
-
 COPY docker-entrypoint.sh /usr/local/bin/
 
 RUN chmod +x /usr/local/bin/docker-entrypoint.sh
 
 ENTRYPOINT ["/usr/local/bin/docker-entrypoint.sh"]
 CMD ["apache2-foreground"]
+
+EXPOSE 80
```
Comment (bucky-boy) [inline docker/Dockerfile]: Exposing two ports?

File: yoti/views/admin-options.php [hunk 4]
```diff
@@ -15,6 +15,7 @@
     <h1>Yoti Settings</h1>
     <p>You need to first create a Yoti App at <?php echo $dashboardLink ?>.</p>
     <p>Note: On the Yoti Dashboard the callback URL should be set to: <code><?php echo site_url('wp-login.php?yoti-select=1&action=link', 'https'); ?></code></p>
+    <p>Warning: User IDs provided by Yoti are only valid within the scope of the application. Changing your Yoti application will result in different Yoti user IDs.</p>
```
Comment (bucky-boy) [inline yoti/views/admin-options.php]: Please consider the text from the other PR

File: docker/Dockerfile (hunk 3 above)
Comment (sidibos) [reply docker/Dockerfile]: I am exposing port 80 for http, but can disable it.

Comment (bucky-boy) [reply docker/Dockerfile]: If it only works over https, then I'd just keep that one

File: README.md (hunk 2 above)
Comment (sidibos) [reply README.md:78]: We use different port to avoid conflict when running the containers simultaneously.

**Review by qzagarese — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (bucky-boy) [reply README.md:78]: ok

**Review by bucky-boy — APPROVED** [review]

**Pull request merged**